great_expectations/rule_based_profiler/profiler.py

```diff
@@ -53,8 +52,10 @@ def __init__(
 
         self._rules = []
 
+        rules_configs: Dict[str, Dict] = profiler_config.get("rules", {})
```

Comment:
Logic is the same as above for this and `variables_config`. Makes sense to return an empty dict if we don't have part of the config but I could see an argument made for the default `None`.

Reply:
I think this is good for now (it will save us a bit of error handling so we don't accidentally try and loop over a None), though we may want to change this in the future if we decide to raise an error if the config is empty.

Reply:
Do you think it makes sense to leave a TODO or is it okay as is?

Reply:
OK as is - it doesn't feel like a must-do.